keps/sig-api-machinery/20190425-crd-conversion-webhook.md

```diff
@@ -249,6 +249,14 @@ type ConversionResponse struct {
 
 If the conversion is failed, the webhook should fail the HTTP request with a proper error code and message that will be used to create a status error for the original API caller.
 
+### Metadata
+
+To ensure consistent behaviour of converted CRD following the established API machinery semantics, we will restrict which fields of `ObjectMeta` a conversion webhook can change to
+
+* labels
+* annotations.
+
```

Comment:
Also add the list of fields which must be preserved as-is: name, namespace, kind, uid